aws-dual-stack/README.md

```diff
@@ -0,0 +1,68 @@
+# How to create a new dual-stack cluster in aws
+
+### Prepare for installation
```

Comment:
What requirements do we have for them?
1. jq?
2. yq?
3. awscli?

Reply:
If we can fail if they don't have them we should

Reply:
```bash
if ! command -v kubectl --help &> /dev/null
then
    echo "kubectl could not be found"
    exit 1
fi
```

Reply:
We should also mention what environment variables you expect in the environment.